src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/mobile/register_app_interface_request.cc

```diff
@@ -1171,36 +1176,20 @@ mobile_apis::Result::eType RegisterAppInterfaceRequest::CheckWithPolicyData() {
     }
   }
 
-  mobile_apis::Result::eType result = mobile_apis::Result::SUCCESS;
+  auto result = mobile_apis::Result::SUCCESS;
 
   // If AppHMIType is not included in policy - allow any type
```

Comment:
Noting this comment. If appHMIType is not included in the app_policy in the policy table then all types should be allowed. Adding the exception that WEB_VIEW must exist in the app policy is overcomplicating the logic implemented in the ProcessingAppHMITypesPolicies/ProcessingAppHMITypesInMessage methods.